applications/MeshingApplication/custom_python/add_meshers_to_python.cpp

```diff
@@ -146,6 +148,19 @@ void TriRegenerateMeshGLASS(TriGenGLASSModeler& Mesher, char* ElementName, char*
 
 
 
+///////////////////////////////////////////////////////////////////////////////////////////
+//											//
+//				ADAPTIVE 2D MESHER specifically for DROPLET FORMING 	//
+//											//
+//////////////////////////////////////////////////////////////////////////////////////////
+
+void TriRegenerateMeshDROPLET(TriGenDropletModeler& Mesher, char* ElementName, char* ConditionName, ModelPart& model_part,NodeEraseProcess& node_erase, bool rem_nodes, bool add_nodes, double alpha_shape, double h_factor )
```

Comment:
Are you sure that this function declaration must be placed in this file? Probably you can set up an utility or process and then just implement the Python exportation in this file.

Reply:
I believe that this is done fine.